src/ducks/index.js

```diff
@@ -107,9 +107,3 @@ export const getTransactions = state => {
 export const getPendingTransactions = state => {
 	return state.transaction.pendingTransactions;
 };
-
-// LOANS REDUCERS
-
-export const getLoans = state => state.loans.loans;
-
-export const getIsFetchingLoans = state => state.loans.isFetchingLoans;
```

Comment:
moving selectors to sit in their "duck" folder.

Reply:
Yeah, we put everything in the duck file, and maintain everything from there. This is handy when we refactor the code, its easy to see which selectors will be affected.

Reply:
yeah agreed